package.json

```diff
@@ -50,6 +51,9 @@
     "tape-async": "^2.3.0",
     "tmp": "0.0.33"
   },
+  "peerDependencies": {
+    "@types/firefox-webext-browser": ">=58.0.0"
```

Comment:
I don't think this is a good idea. Most people don't need these types and `peerDependencies` won't be installed automatically, resulting in a warning.

Either it's a dependency or it's not.

Reply:
For those don't need it, we should not force them to install and they can ignore the warning if their project work without types. For those who need it, we should give them a sign (Hey look like you missing something you might need to install).